core/src/main/java/com/sequenceiq/cloudbreak/service/KubernetesConfigService.java

```diff
@@ -70,26 +43,22 @@ public KubernetesConfig getByNameForWorkspace(String name, Workspace workspace)
     }
 
     @Override
-    public Set<Cluster> getClustersUsingResource(KubernetesConfig resource) {
-        return Collections.emptySet();
+    protected WorkspaceResourceRepository<KubernetesConfig, Long> repository() {
+        return kubernetesConfigRepository;
     }
 
     @Override
-    public Set<Cluster> getClustersUsingResourceInEnvironment(KubernetesConfig kubernetesConfig, Long environmentId) {
-        return Collections.emptySet();
+    protected void prepareDeletion(KubernetesConfig kubernetesConfig) {
```

Comment:
This method has a proper implementation in the previously extended abstract class `AbstractEnvironmentAwareService`
Please implement the same logic!